diff -Naur --exclude='*~' hol/tools/smart-configure.sml hol.fink/tools/smart-configure.sml
--- hol/tools/smart-configure.sml	2005-09-08 21:48:13.000000000 -0700
+++ hol.fink/tools/smart-configure.sml	2007-11-12 08:59:49.000000000 -0800
@@ -78,30 +78,7 @@
 
 determining "mosmldir";
 
-val mosmldir = let
-  val libdir = hd (!Meta.loadPath)
-  val {arcs, isAbs, vol} = Path.fromString libdir
-  val _ = isAbs orelse
-          (print "\n\n*** ML library directory not specified with absolute";
-           print "filename --- aborting\n";
-           Process.exit Process.failure)
-  val (arcs', lib) = frontlast arcs
-  val _ =
-      if lib <> "lib" then
-        print "\nMosml library directory (from loadPath) not .../lib -- weird!\n"
-      else ()
-  val candidate =
-      Path.toString {arcs = arcs' @ ["bin"], isAbs = true, vol = vol}
-  val mosml' = if OS = "winNT" then "mosmlc.exe" else "mosmlc"
-  val _ =
-      if FileSys.access (Path.concat(candidate, mosml'), [FileSys.A_EXEC]) then
-        ()
-      else (print ("\nCouldn't find executable mosmlc in "^candidate^"\n");
-            print ("Giving up - please use config-override file to fix\n");
-            Process.exit Process.failure)
-in
-  candidate
-end;
+val mosmldir = "@PREFIX@/bin" ;
 
 
 determining "holdir";
