diff -Naur --exclude='*~' ocaml-re-ocaml-re-1.2.2/setup.ml ocaml-re-ocaml-re-1.2.2.fink/setup.ml
--- ocaml-re-ocaml-re-1.2.2/setup.ml	2014-05-05 15:51:46.000000000 +0200
+++ ocaml-re-ocaml-re-1.2.2.fink/setup.ml	2014-09-11 15:07:43.000000000 +0200
@@ -739,7 +739,7 @@
   type s = string
 
 
-  type t = string 
+  type t = string
 
 
   type comparator =
@@ -750,7 +750,7 @@
     | VLesserEqual of t
     | VOr of  comparator * comparator
     | VAnd of comparator * comparator
-    
+
 
 
   (* Range of allowed characters *)
@@ -945,17 +945,17 @@
 
 
 
-  type license = string 
+  type license = string
 
 
-  type license_exception = string 
+  type license_exception = string
 
 
   type license_version =
     | Version of OASISVersion.t
     | VersionOrLater of OASISVersion.t
     | NoVersion
-    
+
 
 
   type license_dep_5_unit =
@@ -964,19 +964,19 @@
       excption:  license_exception option;
       version:   license_version;
     }
-    
+
 
 
   type license_dep_5 =
     | DEP5Unit of license_dep_5_unit
     | DEP5Or of license_dep_5 list
     | DEP5And of license_dep_5 list
-    
+
 
   type t =
     | DEP5License of license_dep_5
     | OtherLicense of string (* URL *)
-    
+
 
 
 end
@@ -991,10 +991,10 @@
   open OASISGettext
 
 
-  type test = string 
+  type test = string
 
 
-  type flag = string 
+  type flag = string
 
 
   type t =
@@ -1004,10 +1004,10 @@
     | EOr of t * t
     | EFlag of flag
     | ETest of test * string
-    
 
 
-  type 'a choices = (t * 'a) list 
+
+  type 'a choices = (t * 'a) list
 
 
   let eval var_get t =
@@ -1089,9 +1089,9 @@
     | Para of string
     | Verbatim of string
     | BlankLine
-    
 
-  type t = elt list 
+
+  type t = elt list
 
 end
 
@@ -1102,40 +1102,40 @@
 
 
 
-  type name          = string 
-  type package_name  = string 
-  type url           = string 
-  type unix_dirname  = string 
-  type unix_filename = string 
-  type host_dirname  = string 
-  type host_filename = string 
-  type prog          = string 
-  type arg           = string 
-  type args          = string list 
-  type command_line  = (prog * arg list) 
+  type name          = string
+  type package_name  = string
+  type url           = string
+  type unix_dirname  = string
+  type unix_filename = string
+  type host_dirname  = string
+  type host_filename = string
+  type prog          = string
+  type arg           = string
+  type args          = string list
+  type command_line  = (prog * arg list)
 
 
-  type findlib_name = string 
-  type findlib_full = string 
+  type findlib_name = string
+  type findlib_full = string
 
 
   type compiled_object =
     | Byte
     | Native
     | Best
-    
+
 
 
   type dependency =
     | FindlibPackage of findlib_full * OASISVersion.comparator option
     | InternalLibrary of name
-    
+
 
 
   type tool =
     | ExternalTool of name
     | InternalExecutable of name
-    
+
 
 
   type vcs =
@@ -1148,7 +1148,7 @@
     | Arch
     | Monotone
     | OtherVCS of url
-    
+
 
 
   type plugin_kind =
@@ -1176,7 +1176,7 @@
       ]
 
 
-  type 'a plugin = 'a * name * OASISVersion.t option 
+  type 'a plugin = 'a * name * OASISVersion.t option
 
 
   type all_plugin = plugin_kind plugin
@@ -1188,7 +1188,7 @@
 (* # 115 "src/oasis/OASISTypes.ml" *)
 
 
-  type 'a conditional = 'a OASISExpr.choices 
+  type 'a conditional = 'a OASISExpr.choices
 
 
   type custom =
@@ -1196,7 +1196,7 @@
         pre_command:  (command_line option) conditional;
         post_command: (command_line option) conditional;
       }
-      
+
 
 
   type common_section =
@@ -1205,7 +1205,7 @@
         cs_data: PropList.Data.t;
         cs_plugin_data: plugin_data;
       }
-      
+
 
 
   type build_section =
@@ -1225,7 +1225,7 @@
         bs_byteopt:         args conditional;
         bs_nativeopt:       args conditional;
       }
-      
+
 
 
   type library =
@@ -1236,28 +1236,28 @@
         lib_findlib_parent:     findlib_name option;
         lib_findlib_name:       findlib_name option;
         lib_findlib_containers: findlib_name list;
-      } 
+      }
 
 
   type object_ =
       {
         obj_modules:            string list;
         obj_findlib_fullname:   findlib_name list option;
-      } 
+      }
 
 
   type executable =
       {
         exec_custom:          bool;
         exec_main_is:         unix_filename;
-      } 
+      }
 
 
   type flag =
       {
         flag_description:  string option;
         flag_default:      bool conditional;
-      } 
+      }
 
 
   type source_repository =
@@ -1269,7 +1269,7 @@
         src_repo_branch:      string option;
         src_repo_tag:         string option;
         src_repo_subdir:      unix_filename option;
-      } 
+      }
 
 
   type test =
@@ -1280,7 +1280,7 @@
         test_working_directory:  unix_filename option;
         test_run:                bool conditional;
         test_tools:              tool list;
-      } 
+      }
 
 
   type doc_format =
@@ -1291,7 +1291,7 @@
     | Info of unix_filename
     | DVI
     | OtherDoc
-    
+
 
 
   type doc =
@@ -1307,7 +1307,7 @@
         doc_format:      doc_format;
         doc_data_files:  (unix_filename * unix_filename option) list;
         doc_build_tools: tool list;
-      } 
+      }
 
 
   type section =
@@ -1318,7 +1318,7 @@
     | SrcRepo    of common_section * source_repository
     | Test       of common_section * test
     | Doc        of common_section * doc
-    
+
 
 
   type section_kind =
@@ -1363,7 +1363,7 @@
         disable_oasis_section:  unix_filename list;
         schema_data:            PropList.Data.t;
         plugin_data:            plugin_data;
-      } 
+      }
 
 
 end
@@ -3925,7 +3925,7 @@
                in
                  program_files/(pkg_name ())
            | _ ->
-               "/usr/local")
+               "@DESTDIR@/@PREFIX@")
 
 
   let exec_prefix =
@@ -3979,7 +3979,7 @@
   let datarootdir =
     p "datarootdir"
       (fun () -> s_ "Read-only arch-independent data root")
-      (fun () -> "$prefix"/"share")
+      (fun () -> "@DESTDIR@/@PREFIX@"/"share")
 
 
   let datadir =
@@ -5752,7 +5752,7 @@
       (* The first command does not use -add: *)
       split first_args files
     else
-      ["install" :: findlib_name :: meta :: files]
+      ["install" :: "-destdir" :: "@DESTDIR@" :: findlib_name :: meta :: files]
 
 
   let install pkg argv =
@@ -6257,7 +6257,7 @@
 
 
 
-  type extra_args = string list 
+  type extra_args = string list
 
 
   let ocamlbuild_clean_ev = "ocamlbuild-clean"
@@ -6574,7 +6574,7 @@
     {
       extra_args: string list;
       run_path: unix_filename;
-    } 
+    }
 
 
   let doc_build run pkg (cs, doc) argv =
@@ -6636,7 +6636,7 @@
         cmd_main:      command_line conditional;
         cmd_clean:     (command_line option) conditional;
         cmd_distclean: (command_line option) conditional;
-      } 
+      }
 
 
   let run  = BaseCustom.run
