diff -Naur --exclude='*~' mc-4.7.0.2/contrib/Makefile.am mc-4.7.0.2.fink/contrib/Makefile.am
--- mc-4.7.0.2/contrib/Makefile.am	2009-12-30 09:21:28.000000000 +0100
+++ mc-4.7.0.2.fink/contrib/Makefile.am	2010-02-25 10:48:30.000000000 +0100
@@ -14,7 +14,7 @@
 SCRIPTS_OUT = mc.csh mc.sh mc-wrapper.csh mc-wrapper.sh
 
 pkglibexec_SCRIPTS = $(SCRIPTS_OUT)
-pkglibexecdir =		$(libexecdir)/@PACKAGE@
+pkglibexecdir =		$(libexecdir)
 
 cfgdir = $(sysconfdir)/@PACKAGE@
 cfg_DATA = \
diff -Naur --exclude='*~' mc-4.7.0.2/contrib/Makefile.in mc-4.7.0.2.fink/contrib/Makefile.in
--- mc-4.7.0.2/contrib/Makefile.in	2010-02-02 16:24:33.000000000 +0100
+++ mc-4.7.0.2.fink/contrib/Makefile.in	2010-02-25 10:48:20.000000000 +0100
@@ -304,7 +304,7 @@
 SCRIPTS_IN = mc.csh.in mc.sh.in mc-wrapper.csh.in mc-wrapper.sh.in
 SCRIPTS_OUT = mc.csh mc.sh mc-wrapper.csh mc-wrapper.sh
 pkglibexec_SCRIPTS = $(SCRIPTS_OUT)
-pkglibexecdir = $(libexecdir)/@PACKAGE@
+pkglibexecdir = $(libexecdir)
 cfgdir = $(sysconfdir)/@PACKAGE@
 cfg_DATA = \
 	$(LIBFILES_ADD)
diff -Naur --exclude='*~' mc-4.7.0.2/src/Makefile.in mc-4.7.0.2.fink/src/Makefile.in
--- mc-4.7.0.2/src/Makefile.in	2010-02-02 16:24:53.000000000 +0100
+++ mc-4.7.0.2.fink/src/Makefile.in	2010-02-25 10:45:18.000000000 +0100
@@ -368,7 +368,7 @@
 	-DLOCALEDIR=\""$(localedir)"\" \
 	-DSYSCONFDIR=\""$(sysconfdir)/@PACKAGE@/"\" $(am__append_3)
 AM_CFLAGS = $(GLIB_CFLAGS) -I$(top_srcdir) $(PCRE_CFLAGS)
-pkglibexecdir = $(libexecdir)/@PACKAGE@
+pkglibexecdir = $(libexecdir)
 mcmfmt_SOURCES = mfmt.c
 man2hlp_SOURCES = man2hlp.c
 man2hlp_LDADD = \
diff -Naur --exclude='*~' mc-4.7.0.2/src/consaver/Makefile.am mc-4.7.0.2.fink/src/consaver/Makefile.am
--- mc-4.7.0.2/src/consaver/Makefile.am	2010-01-29 09:09:28.000000000 +0100
+++ mc-4.7.0.2.fink/src/consaver/Makefile.am	2010-02-25 10:45:41.000000000 +0100
@@ -1,4 +1,4 @@
-pkglibexecdir =		$(libexecdir)/@PACKAGE@
+pkglibexecdir =		$(libexecdir)
 
 pkglibexec_PROGRAMS = cons.saver
 
diff -Naur --exclude='*~' mc-4.7.0.2/src/consaver/Makefile.in mc-4.7.0.2.fink/src/consaver/Makefile.in
--- mc-4.7.0.2/src/consaver/Makefile.in	2010-02-02 16:24:54.000000000 +0100
+++ mc-4.7.0.2.fink/src/consaver/Makefile.in	2010-02-25 10:46:15.000000000 +0100
@@ -298,7 +298,7 @@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-pkglibexecdir = $(libexecdir)/@PACKAGE@
+pkglibexecdir = $(libexecdir)
 cons_saver_SOURCES = cons.saver.c
 cons_saver_CFLAGS = -I$(top_srcdir) $(GLIB_CFLAGS) $(PCRE_CFLAGS)
 cons_saver_LDADD = $(GLIB_LIBS) $(PCRE_LIBS)
